test/unit/annotation_spec.js

```diff
@@ -343,6 +411,154 @@ describe('annotation', function() {
+        { ref: annotationRef, data: annotationDict, },
+        { ref: replyRef, data: replyDict, }
+      ]);
+
+      AnnotationFactory.create(xref, replyRef, pdfManagerMock,
+        idFactoryMock).then(({ data, }) => {
+          expect(data.inReplyTo).toBeDefined();
+          expect(data.replyType).toEqual('Group');
+          expect(data.contents).toEqual('TestText');
+
+          done();
+        }, done.fail);
+    });
+
+    it('should correctly parse IRT, and Reply RT with reply text',
+        function (done) {
+      const annotationDict = new Dict();
+      annotationDict.set('Type', Name.get('Annot'));
+      annotationDict.set('Subtype', Name.get('Text'));
+      annotationDict.set('Contents', 'TestText');
+      const annotationRef = new Ref(819, 0);
+
+      const replyDict = new Dict();
+      replyDict.set('Type', Name.get('Annot'));
+      replyDict.set('Subtype', Name.get('Text'));
+      replyDict.set('IRT', annotationDict);
+      replyDict.set('RT', Name.get('R'));
+      replyDict.set('Contents', 'ReplyText');
+
+      const replyRef = new Ref(820, 0);
+
+      const xref = new XRefMock([
+        { ref: annotationRef, data: annotationDict, },
+        { ref: replyRef, data: replyDict, }
+      ]);
+
+      AnnotationFactory.create(xref, replyRef, pdfManagerMock,
+        idFactoryMock).then(({ data, }) => {
+          expect(data.inReplyTo).toBeDefined();
+          expect(data.replyType).toEqual('R');
+          expect(data.contents).toEqual('ReplyText');
+          done();
+        }, done.fail);
+    });
+
+    it('should correctly parse IRT, Group RT and ' +
+        'popup of Group annot should take contents of group master',
+        function (done) {
+      const annotationDict = new Dict();
+      annotationDict.set('Type', Name.get('Annot'));
+      annotationDict.set('Subtype', Name.get('Text'));
+      annotationDict.set('Contents', 'Correct Text');
+      const annotationRef = new Ref(822, 0);
+
+      const replyDict = new Dict();
+      replyDict.set('Type', Name.get('Annot'));
+      replyDict.set('Subtype', Name.get('Text'));
+      replyDict.set('IRT', annotationDict);
+      replyDict.set('RT', Name.get('Group'));
+      replyDict.set('Contents', 'Reply Text');
+      const replyRef = new Ref(823, 0);
+
+      const popupDict = new Dict();
+      popupDict.set('Type', Name.get('Annot'));
+      popupDict.set('Subtype', Name.get('Popup'));
+      popupDict.set('Parent', replyDict);
+      popupDict.set('Contents', 'Wrong text');
+      const popupRef = new Ref(824, 0);
+
+      const xref = new XRefMock([
+        { ref: annotationRef, data: annotationDict, },
+        { ref: replyRef, data: replyDict, },
+        { ref: popupRef, data: popupDict, }
+
+      ]);
+
+      AnnotationFactory.create(xref, popupRef, pdfManagerMock,
+        idFactoryMock).then(({ data, }) => {
+
+          expect(data.contents).toEqual('Correct Text');
+
+          done();
+        }, done.fail);
+    });
```

Comment:
Could you move this last test to the `PopupAnnotation`'s `describe` block at:
https://github.com/mozilla/pdf.js/blob/6e0e725af401cb3a7a96aba15624b1c625967cc6/test/unit/annotation_spec.js#L1674

That's a better place for this test since it's relevant in that scope.